build-tests/api-documenter-test/etc/api-documenter-test.api.json

```diff
@@ -1471,6 +1475,157 @@
           "releaseTag": "Public",
           "name": "IDocInterface6",
           "members": [
+            {
+              "kind": "PropertySignature",
+              "canonicalReference": "api-documenter-test!IDocInterface6#arrayProperty:member",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Reference",
+                  "text": "arrayProperty"
+                },
+                {
+                  "kind": "Content",
+                  "text": ": "
+                },
+                {
+                  "kind": "Reference",
+                  "text": "IDocInterface1",
+                  "canonicalReference": "api-documenter-test!IDocInterface1:interface"
+                },
+                {
+                  "kind": "Content",
+                  "text": "[]"
+                },
+                {
+                  "kind": "Content",
+                  "text": ";"
+                }
+              ],
+              "releaseTag": "Public",
+              "name": "arrayProperty",
+              "propertyTypeTokenRange": {
+                "startIndex": 2,
+                "endIndex": 4
+              }
+            },
+            {
+              "kind": "MethodSignature",
+              "canonicalReference": "api-documenter-test!IDocInterface6#genericReferenceMethod:member(1)",
+              "docComment": "",
+              "excerptTokens": [
+                {
+                  "kind": "Reference",
+                  "text": "genericReferenceMethod"
+                },
+                {
+                  "kind": "Content",
+                  "text": "<"
+                },
+                {
+                  "kind": "Reference",
+                  "text": "T"
+                },
+                {
+                  "kind": "Content",
+                  "text": ">("
+                },
+                {
+                  "kind": "Reference",
+                  "text": "x"
+                },
+                {
+                  "kind": "Content",
+                  "text": ": "
+                },
+                {
+                  "kind": "Reference",
+                  "text": "T",
+                  "canonicalReference": "T"
```

Comment:
This corresponds to `genericReferenceMethod<T>(x: T): T;` from `IDocInterface6`.

Is it intentional to generate a reference for a type parameter?  That seems not useful.

Reply:
I can remove the canonical reference.